src/colorizer/ColorizeCanvas.ts

```diff
@@ -114,14 +133,33 @@ export default class ColorizeCanvas {
     this.pickScene = new Scene();
     this.pickScene.add(this.pickMesh);
 
+    // Configure lines
+    this.points = new Float32Array([0, 0, 0]);
+
+    this.lineGeometry = new BufferGeometry();
+    this.lineGeometry.setAttribute("position", new BufferAttribute(this.points, 3));
+    // Line material has its own vertex + fragment shader because they're
+    // required to make sure they resize correctly with the frame. This is because
+    // the points are drawn in a relative canvas range [-1, 1] rather than the frame.
+    this.lineMaterial = new LineBasicMaterial({
```

Comment:
I think the comment above here is basically obsolete

Reply:
oops good catch